.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,71 @@
+---
+name: Bug Template
+about: Create a bug report to help us improve
+title: ''
+labels: 'bug, new issue'
```

Comment:
lets remove "new issue" as we wont be using it in this repository for the time being